apps/files/lib/Search/FilesSearchProvider.php

```diff
@@ -110,11 +110,13 @@ public function search(IUser $user, ISearchQuery $query): SearchResult {
 					? $this->urlGenerator->linkToRouteAbsolute('core.Preview.getPreviewByFileId', ['x' => 32, 'y' => 32, 'fileId' => $result->id])
 					: '';
 
-				return new SearchResultEntry(
+				return new ObjectSearchResultEntry(
 					$thumbnailUrl,
 					$result->name,
 					$this->formatSubline($result),
 					$this->urlGenerator->getAbsoluteURL($result->link),
+					'file',
```

Comment:
we use plurals in comments table, activity and others.
Maybe here too?

```suggestion
					'files',
```